### PR TITLE
 Added an action and reducer to accord for deleting a markdown

### PR DESCRIPTION
**Pull request by Zilula**
Added an action and reducer case that accounts for a deletion of a markdown file. Added a button to fire a function that calls said action and update the current store/state

### CODE REVIEW

File: src/components/SavedMarkdownList.js
Comment (acrankin) [inline src/components/SavedMarkdownList.js]:
```suggestion
```

File: src/components/SavedMarkdownList.js
Comment (acrankin) [inline src/components/SavedMarkdownList.js]:
```suggestion
```

File: src/selectors/markdownListSelectors.js
Comment (acrankin) [inline src/selectors/markdownListSelectors.js:10]: No commented code!

File: src/reducers/markdownListReducer.js
Comment (MariahAdams) [inline src/reducers/markdownListReducer.js:1]: I would add a space after this import to keep syntax consistent across files.

**Review by MariahAdams — APPROVED** [review]
You're good to go once you clean up the comments on the PR!

**Review by caripizza — APPROVED** [review]
After confusing merge conflicts we are going to add the updates per Reviewers' comments AFTER we merge this branch in (those will come in a new branch/pr).

**Pull request merged**
